src/elements/pages/dev/games.ts

```diff
@@ -267,10 +267,10 @@ export default class DevGames extends LitElement {
 				  </div>`
 				: null}
 			${when(
-				!config.IS_PROD && global.currentIdentity.isRegistered,
+				global.currentIdentity.isRegistered,
 				() =>
 					html` <div
-						class="dashed-border-button flex justify-center items-center hover:cursor-pointer w-full h-32 place-content-center text-[#d1d1d1]  hover:text-white hover:bg-[#ffffff05]"
+						class="dashed-border-button flex justify-center items-center hover:cursor-pointer w-full h-32 place-content-center text-[#d1d1d1]  hover:text-white hover:bg-button-bg-hover-color"
```

Comment:
If you're converting colors that we had before, I'd recommend picking the nearest Tailwind native color instead.